src/conduit/components/forms.cljs

```diff
@@ -22,17 +22,21 @@
              v))])])
 
 (rum/defc InputField
-  [{:keys [placeholder type value errors on-blur on-focus on-change]}]
-  [:fieldset.form-group
-   [:input.form-control.form-control-lg
-    {:placeholder placeholder
-     :on-change #(on-change (.. % -target -value))
-     :on-blur on-blur
-     :on-focus on-focus
-     :type (or type :text)
-     :value value}]
-   (when errors
-     (InputErrors errors))])
+  [{:keys [placeholder type value errors on-blur on-focus on-change field-type]}]
+  (let [input-field (case field-type
+                :textarea (fn [attrs & children]
```

Comment:
It's better to not mix different input fields in a one component, create a separate component for `textarea`.